src/App.js

```diff
@@ -1,18 +1,15 @@
 import React from 'react';
-import { Routes, Route } from 'react-router-dom';
+import { Provider } from 'react-redux';
+import store from './redux/configureStore';
 import Books from './Pages/Books';
-import Categories from './Pages/Categories';
-import NavBar from './components/NavBar';
 import './App.css';
 
-export default function App() {
+function App() {
   return (
-    <div>
-      <NavBar />
-      <Routes>
-        <Route path="/" element={<Books />} />
-        <Route path="/categories" element={<Categories />} />
```

Comment:
- [ ] The nav bar is not existing now on the page, So I can't test the` check status` button on the categories page. Please fix that.